osf/metrics.py

```diff
@@ -124,6 +124,10 @@ class BasePreprintMetric(MetricMixin, metrics.Metric):
     preprint_id = metrics.Keyword(index=True, doc_values=True, required=True)
     version = metrics.Keyword(index=True, doc_values=True)
     path = metrics.Text(index=True)
+    referrer = metrics.Text(index=True)
+    user_agent = metrics.Text(index=True)
+    origin = metrics.Text(index=True)
+    uri = metrics.Text(index=True)
```

Comment:
Is there any way to backfill this data? user_agent, orgin, uri, referrer? Otherwise, we just have it going forward.

Reply:
No, all this info is new.